frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -50,6 +51,23 @@ export const getOCSRequestData = (
     },
   } as K8sResourceKind;
 
+  if (isMinimal) {
+    requestData.spec = {
+      resources: {
+        mds: {
+          limits: {
+            cpu: '3',
+            memory: '8Gi',
+          },
+          requests: {
+            cpu: '1',
+            memory: '8Gi',
+          },
+        },
+      },
+    };
+  }
```

Comment:
Have you checked this with OCS backend team?

Reply:
yes. I and @oritwas talked about this. For now, we are moving forward with this. Tested too and is successfully getting deployed.